maxim/logger/gemini/async_client.py

```
@@ -119,6 +119,33 @@ async def send_message(
+                            self._trace_id,
+                            {
+                                "id": tool_call_id,
+                                "name": tool_name,
+                                "description": "Gemini tool call",
+                                "args": tool_args,
+                            },
+                        )
+                        tool_call.result(
+                            "Executed via Gemini automatic function calling"
+                        )
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)